source/_components/hdmi_cec.markdown

```diff
@@ -30,13 +30,13 @@ The computer running Home Assistant must support CEC, and of course be connected
 Create a symlink to the `cec` installation.
  
 ```bash
-$ ln -s /usr/local/lib/python3.4/dist-packages/cec /path/to/your/venv/lib/python3.4/site-packages
+$ ln -s /usr/local/lib/python3.4/site-packages/cec /path/to/your/venv/lib/python3.4/site-packages
 ```
 
 For the default virtual environment of a [Raspberry Pi All-In-One installation](/getting-started/installation-raspberry-pi-all-in-one/) the command would be as follows.
 
 ```bash
-$ ln -s /usr/local/lib/python3.4/dist-packages/cec /srv/hass/hass_venv/lib/python3.4/site-packages
+$ ln -s /usr/local/lib/python3.4/site-packages/cec /srv/homeassistant/homeassistant_venv/lib/python3.4/site-packages
```

Comment:
It depends on installation type. So provided values are just for explanation. Everyone has to find his own values.